### PR TITLE
style: make filter popover button smaller

### PR DESCRIPTION
**Pull request by FrantisekMichalSebestyen**
style: leave space for selectionCount
style: stop border from jumping up and down

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (link to Jira or GitHub issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [ ] I have updated the [CHANGELOG](../CHANGELOG.md)
- [ ] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/js/components/catalog-new/NewCustomSelectPopoverLabel.vue [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <template>
-    <span class="tw-font-semibold tw-font-sm md:tw-font-base">
+    <span class="tw-font-semibold tw-font-sm md:tw-font-base tw-min-w-max">
```
Comment (eronisko) [inline resources/js/components/catalog-new/NewCustomSelectPopoverLabel.vue:2]: Why `tw-min-w-max`?

File: resources/js/components/catalog-new/PopoverFilter.vue [hunk 2]
```diff
@@ -1,40 +1,46 @@
...
+                        <svg
+                            v-else
+                            xmlns="http://www.w3.org/2000/svg"
+                            class="tw-w-4 tw-h-4 tw-fill-current"
+                            viewBox="0 0 256 256"
+                        >
+                            <path
+                                d="M216.49,104.49l-80,80a12,12,0,0,1-17,0l-80-80a12,12,0,0,1,17-17L128,159l71.51-71.52a12,12,0,0,1,17,17Z"
+                            ></path>
+                        </svg>
```
Comment (eronisko) [inline resources/js/components/catalog-new/PopoverFilter.vue:39]: Should these be extracted? Are they used elsewhere?

File: resources/js/components/catalog-new/NewCustomSelectPopoverLabel.vue (hunk 1 above)
Comment (FrantisekMichalSebestyen) [reply resources/js/components/catalog-new/NewCustomSelectPopoverLabel.vue:2]: I use it as `white-space: nowrap`here except it is a bit stronger as it prevents all elements from wrapping

File: resources/js/components/catalog-new/PopoverFilter.vue (hunk 2 above)
Comment (FrantisekMichalSebestyen) [reply resources/js/components/catalog-new/PopoverFilter.vue:39]: Is done in #857

**Review by eronisko — APPROVED** [review]
Nice, thanks

**Pull request merged**